timefold-solver-python-core/src/main/python/score/_score_analysis.py

```diff
@@ -103,6 +119,10 @@ class ConstraintMatch(Generic[Score_]):
     def identification_string(self) -> str:
         return self.constraint_ref.constraint_id
 
+    @property
+    def get_indicted_object_list(self):
```

Comment:
Not a good name for an attribute. Use `indicted_object_list` or `indicted_objects`.